src/Beta/Groups.Actions/Groups.Actions/Microsoft.Graph.Groups.Actions.psd1

```diff
@@ -0,0 +1,205 @@
+#
+# Module manifest for module 'Microsoft.Graph.Groups.Actions'
+#
+# Generated by: Microsoft Corporation
+#
+# Generated on: 4/10/2020
+#
+
+@{
+
+# Script module or binary module file associated with this manifest.
+RootModule = './Microsoft.Graph.Groups.Actions.psm1'
+
+# Version number of this module.
+ModuleVersion = '0.2.1'
```

Comment:
i'm using that these just all get regenerated with the new version specified in the modulemetadata file

Reply:
That's right, they will be regenerated and set to the correct module version in their readme.md. For new modules - such as this one, I do check in their `*.psd1` so that we can review the generated cmdlet names.